.gitattributes

```diff
@@ -0,0 +1,3 @@
+*.owl -diff -merge
+nodegroups/ingestion/*/ingest_*.json -diff -merge
+nodegroups/queries/*/query*.json -diff -merge
```

Comment:
I'm in favor of hiding the OWL files.  I've almost never found their diffs to be worth reading. 

I'm fine with hiding the json files too, since they're machine-generated as well.